### PR TITLE
Add Japanese translation of "token rotation" docment

### PR DESCRIPTION
**Pull request by hirosassa**
(Describe the goal of this PR. Mention any related Issue numbers)

Related to #414 
I added Japanese translation of `Token rotation` document introduced in #413.

### Category (place an `x` in each of the `[ ]`)

* [ ] `slack_bolt.App` and/or its core components
* [ ] `slack_bolt.async_app.AsyncApp` and/or its core components
* [ ] Adapters in `slack_bolt.adapter`
* [x] Document pages under `/docs`
* [ ] Others

## Requirements (place an `x` in each `[ ]`)

Please read the [Contributing guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and [Code of Conduct](https://slackhq.github.io/code-of-conduct) before creating this issue or pull request. By submitting, you are agreeing to those rules.

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/bolt-python/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).
* [x] I've run `./scripts/install_all_and_run_tests.sh` after making the changes.

### CODE REVIEW

File: docs/_advanced/ja_token_rotation.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+title: トークンのローテーション
+lang: ja-jp
+slug: token-rotation
+order: 6
+---
+
+<div class="section-content">
+Bolt for Python [v1.7.0](https://github.com/slackapi/bolt-python/releases/tag/v1.7.0) より、[OAuth V2 RFC](https://datatracker.ietf.org/doc/html/rfc6749#section-10.4) で規定され、アクセストークンに対するセキュリティを強化する、トークンローテーションを提供しています。
+
+既存の Slack app のアクセストークンが無期限に存在し続けるのに対し、トークンローテーションを有効にすると、アクセストークンは失効します。これに対し、リフレッシュトークンを用いて、長期にわたりアクセストークンの更新行います。
```
Comment (seratch) [inline docs/_advanced/ja_token_rotation.md:11]: * We use "Slack アプリ" for Slack app
* "トークンローテーションを有効にしたアプリでは" is not a directly translated phrase but it should be easier to understand for anyone
* No need to have "これに対し、"
```suggestion
既存の Slack アプリではアクセストークンが無期限に存在し続けるのに対して、トークンローテーションを有効にしたアプリではアクセストークンが失効するようになります。リフレッシュトークンを利用して、アクセストークンを長期間にわたって更新し続けることができます。
```

File: docs/_advanced/ja_token_rotation.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+Bolt for Python は[組み込みの OAuth](https://slack.dev/bolt-python/concepts#authenticating-oauth) が使用されている場合、自動的にトークンローテーションを行います。
```
Comment (seratch) [inline docs/_advanced/ja_token_rotation.md:13]: Your translation here is already great enough but we can make the sentence a bit clearer this way:

```suggestion
[Bolt for Python の組み込みの OAuth 機能](https://slack.dev/bolt-python/concepts#authenticating-oauth) を使用していれば、Bolt for Python が自動的にトークンローテーションの処理をハンドリングします。
```

File: docs/_advanced/ja_token_rotation.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+トークンローテーションについての詳細な情報は、この[ドキュメント](https://api.slack.com/authentication/rotation)をご覧ください。
```
Comment (seratch) [inline docs/_advanced/ja_token_rotation.md:15]: For consistency with other pages:
```suggestion
トークンローテーションに関する詳細は [API ドキュメント](https://api.slack.com/authentication/rotation)を参照してください。
```

**Review by seratch — CHANGES_REQUESTED** [review]
To complete this work, may I ask you to update the "order:" numbers as the English document pull request does: https://github.com/slackapi/bolt-python/pull/413/files

**Comment by seratch** [comment]
@hirosassa Thanks for taking the time to make this pull request! Can you check my review comments?

**Comment by hirosassa** [comment]
@seratch Thanks for your review! I applied suggested changes.

**Review by seratch — APPROVED** [review]
Thanks for the quick update. Looks great to me 👍

**Pull request merged**